stream_alert/alert_processor/main.py

```diff
@@ -40,78 +42,52 @@ def handler(event, context):
             indicates if sending was successful and the second value is the
             output configuration info (ie - 'slack:sample_channel')
     """
-    records = event.get('Records', [])
-    LOGGER.info('Running alert processor for %d records', len(records))
-
-    # A failure to load the config will log the error in load_output_config and return here
+    # A failure to load the config will log the error in load_output_config
+    # and return here
     config = _load_output_config()
     if not config:
         return
 
     region = context.invoked_function_arn.split(':')[3]
     function_name = context.function_name
 
-    status_values = []
-
-    for record in records:
-        sns_payload = record.get('Sns')
-        if not sns_payload:
-            continue
-
-        sns_message = sns_payload['Message']
-        try:
-            loaded_sns_message = json.loads(sns_message)
-        except ValueError as err:
-            LOGGER.error('An error occurred while decoding message to JSON: %s', err)
-            continue
+    # Return the current list of statuses back to the caller
+    return list(status for status in run(event, region, function_name, config))
```

Comment:
You can simplify this further:
```python
return list(run(event, region, function_name, config))
```
`list` will automatically transform the generator into a list by iterating through it; you don't have to manually iterate